src/components/query-viewer/controller/query-viewer-controller.tsx

```diff
@@ -60,7 +60,19 @@ export class QueryViewerController {
   /**
    * Base URL of the server
    */
-  @Prop() readonly baseUrl: string;
+  @Prop() readonly baseUrl = process.env.GENEXUS_QUERY_URL;
+  /**
+   * Authentication API Key
+   */
+  @Prop() readonly apiKey = process.env.GENEXUS_API_KEY;
+  /**
+   * Authentication Saia Token
+   */
+  @Prop() readonly saiaToken = process.env.GENEXUS_SAIA_TOKEN;
+  /**
+   * Optional Saia user ID
+   */
+  @Prop() readonly saiaUserId = process.env.GENEXUS_SAIA_USER_ID;
```

Comment:
In a CSR only framework, this solution does not seem correct. Can we rethink this?